orc8r/cloud/helm/orc8r/charts/secrets/values.yaml

```diff
@@ -55,6 +63,10 @@ secret:
         elasticPort: 9200
       analytics.yml: |-
         analysisSchedule: 3
+      nprobe.yml: |-
+        operatorID: 1
+        updateIntervalSecs: 60
+        backoffIntervalSecs: 360
```

Comment:
Use the `nprobe.yml` file to set these default values. Only include values here that *must* be set (i.e. are ~unique per deployment / a default doesn't make sense)

That said, if you're providing defaults for all these values in the tf, then don't include the configs here at all